configure.ac

```diff
@@ -357,8 +355,7 @@ AC_ARG_ENABLE(optimizations,
 AC_ARG_ENABLE(debug-cbdata,
   AS_HELP_STRING([--enable-debug-cbdata],
       [Provide some debug information in cbdata]), [
-SQUID_YESNO([$enableval],
-    [unrecognized argument to --enable-debug-cbdata: $enableval])
+  SQUID_YESNO([$enableval],[--enable-debug-cbdata])
```

Comment:
There is a lot of code duplication in all these blobs. I wonder whether SQUID_YESNO could automatically extract or reconstruct the name of the option from some variable set by AC_ARG_ENABLE and AC_ARG_WITH (e.g., the variable that holds their parameter). This is just a comment. I do not insist on any changes here because it may not be feasible (or may not be a good idea) to remove this duplication.

Reply:
Aye, part of the aim of these cleanups is to see how much of the unnecessary uniqueness can be removed. In hopes that someone can eventually write a macro and hide the M4 verbosity/duplication.